cactus-git/src/main/java/com/telenav/cactus/git/GitCheckout.java

```diff
@@ -272,7 +269,7 @@ public static Optional<GitCheckout> submodulesRoot(Path dirOrFile)
     private final GitCommand<Optional<ZonedDateTime>> commitDate
             = new GitCommand<>(ProcessResultConverter.strings().trimmed()
                     .map(this::fromGitLogFormat),
-                    "--no-pager", "log", "-1", "--format=format:%cd",
```

Comment:
huh. what was the problem with --no-pager? and don't you want --no-transfer-progress?

Reply:
I added `--no-pager` in `GitCommand` so it is prepended to *every* git command line, since it is an option that all commands accept.

I'm not sure that `--no-transfer-progress` is something that is legal in all commands, right after `git`.